File: Java/src/Neon_Number.java
/**
 * Neon Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-024
 */
import java.util.*;
public class Neon_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Neon Number Checker\n~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        int s=0,d,m=n*n;
        while(m>0)
        {
            d=m%10;
            s=s+d;
            m=m/10;
        }
        if(s==n)
            System.out.print(n+" is a Neon number.");
        else
            System.out.print(n+" is not a Neon number.");        
    }
}

File: Python/Neon_Number.py
print("Neon Number Checker\n")
n = int(input("Enter the number : "))
sq = n * n
s = 0
for digit in str(n) :
	s += int(digit)	
if s == n:
	print(f"{n} is a neon number.")
else:
	print(f"{n} is not a neon number.")